stanford/playbooks/rabbitmq.yml

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: rabbitmq
+    CLUSTER_ELBS:
+      - name: "{{ CLUSTER_DEPLOYMENT }}-elb-{{ CLUSTER_NAME }}"
+        health_check:
+          ping_protocol: tcp
+          ping_port: 5672
+          response_timeout: 5  # seconds
+          interval: 30  # seconds
+          unhealthy_threshold: 2  # attempts
+          healthy_threshold: 10  # attempts
+        listeners:
+          - protocol: tcp
+            load_balancer_port: 5672
+            instance_port: 5672
+          - protocol: tcp
+            load_balancer_port: 6163
+            instance_port: 6163
+        subnets: "{{ CLUSTER_ELB_SUBNETS }}"
+      - name: "{{ CLUSTER_DEPLOYMENT }}-elb-{{ CLUSTER_NAME }}-www"
+        health_check:
+          ping_protocol: http
+          ping_path: '/'
+          ping_port: 15672
+          response_timeout: 5  # seconds
+          interval: 30  # seconds
+          unhealthy_threshold: 2  # attempts
+          healthy_threshold: 10  # attempts
+        listeners:
+          - protocol: https
+            load_balancer_port: 443
+            instance_port: 15672
+            ssl_certificate_id: "{{ DEPLOYMENT_SSL_CERTIFICATE_ID }}"
+        subnets: "{{ CLUSTER_ELB_SUBNETS }}"
+# - hosts: tag_Deployment_stage:&tag_Cluster_rabbitmq:&tag_todo_deploy
```

Comment:
Should we remove this line now?
As written, this means we're always deploying to a fresh base box.